reconcile/utils/terrascript_client.py

```diff
@@ -2450,6 +2454,48 @@ def populate_tf_resource_s3_cloudfront(self, resource, namespace_info):
             '${' + cf_oai_tf_resource.id + '}'
         tf_resources.append(Output(output_name_0_13, value=output_value))
 
+        # aws_s3_bucket_acl
+        if cf_distribution_tf_resource.logging_config:
+            values = {}
+            values['name'] = identifier
```

Comment:
you should probably be using the information from `logging_config`?

Reply:
i think this is fine now, right @maorfr ?